src/main/java/net/revelc/code/formatter/model/ConfigReader.java

```diff
@@ -47,11 +47,10 @@ public Map<String, String> read(final InputStream input) throws IOException, SAX
         digester.addRuleSet(new RuleSet());
 
         final var result = digester.parse(input);
-        if (!(result instanceof Profiles)) {
+        if (!(result instanceof final Profiles profiles)) {
```

Comment:
This is nice. Is this a newer JDK feature?

Reply:
https://docs.oracle.com/en/java/javase/17/language/pattern-matching-instanceof-operator.html

Reply:
I believe sonarlint calls that out and has auto fix for it.

Reply:
The negation makes the scope where the pattern variable can be used a bit less intuitive, but overall, I like it.